v1/storage/internal/errors/errors.go

```diff
@@ -20,7 +20,11 @@ func NewNotFoundError(path storage.Path) *storage.Error {
 }
 
 func NewNotFoundErrorWithHint(path storage.Path, hint string) *storage.Error {
-	return NewNotFoundErrorf("%v: %v", path.String(), hint)
+	message := path.String() + ": " + hint
+	return &storage.Error{
+		Code:    storage.NotFoundErr,
+		Message: message,
+	}
```

Comment:
This is a performance optimization?

Reply:
ok, saw the benchtest 👍

Reply:
Yeah, it's quite insidious how much Sprintf can cost even in simple cases!